src/core/proto/mem_buf_desc.h

```diff
@@ -69,7 +69,12 @@ struct timestamps_t {
  */
 class mem_buf_desc_t {
 public:
-    enum flags { TYPICAL = 0, CLONED = 0x01, ZCOPY = 0x02 };
+    enum flags {
+        TYPICAL = 0,
+        CLONED = 0x01,
+        ZCOPY = 0x02,
+        INVALID = 0x04,
```

Comment:
I would rename it to something more specific than just invalid. Maybe: HAD_CQE_ERROR